### PR TITLE
Ana+Fabo/4280 optimize validator search with tentative store

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Just opening so Fabo can have a look at a performance issue

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
Fixed all performance issues. Requests are instant when data is ready.

**Comment by faboweb** [comment]
🚀 Deployed on https://5eea87b07a390d19f72bfbd7--lunieio.netlify.app

File: api/lib/resolvers.js [hunk 1]
```diff
@@ -47,77 +46,21 @@ function localStore(dataSources, networkId) {
...
   if (activeOnly) {
     validators = validators.filter(({ status }) => status === 'ACTIVE')
   }
-  const validatorInfoMap = await getValidatorInfosMap(
-    localStore(dataSources, networkId)
-  )
-  validators = validators.map((validator) =>
-    enrichValidator(validatorInfoMap[validator.operatorAddress], validator)
-  )
-
```
Comment (Bitcoinera) [inline api/lib/resolvers.js:120]: Uuuh nice! So we do all this ordering and filtering in store now

**Comment by Bitcoinera** [comment]
Really nice this PR! Great job 👏 !

File: api/lib/resolvers.js (hunk 1 above)
Comment (faboweb) [reply api/lib/resolvers.js:120]: enriching is happening once instead of every time

Comment (Bitcoinera) [reply api/lib/resolvers.js:120]: Lovely!!

**Pull request merged**